main.tex

```diff
@@ -1300,7 +1300,16 @@ \section{Conclusion}
 develop, and contribute to those tools within a conscientious and welcoming
 community.
 
-\acknowledgments The \astropy community is supported by and makes use
+\acknowledgments 
+
+We would like to thank the community that have contributed to \astropy, 
```

Comment:
have or has? Community would be singular, but do you want to say "those members of the community"? I'm a little confused and leave that up to a native speaker.